templates/datapipeline.yml

```diff
@@ -0,0 +1,308 @@
+---
+AWSTemplateFormatVersion: 2010-09-09
+Description: DataPipeline EFS Backup DataPipeline Template
+
+Parameters:
+  myS3BackupsBucket:
```

Comment:
We can't implement this replace because of restriction
https://github.com/cloudposse/tf_efs_backup/pull/1#discussion_r131442017
```
Error applying plan:

2 error(s) occurred:
* aws_cloudformation_stack.datapipeline[1]: 1 error(s) occurred:
* aws_cloudformation_stack.datapipeline.1: 
UPDATE_ROLLBACK_COMPLETE: ["Pipeline Definition failed to validate because of following Errors: [{ObjectId = 'tfInstanceType', errors = [Parameter id must have prefix 'my'
```